ui/app/templates/vault/cluster/secrets/backends.hbs

```diff
@@ -78,25 +79,24 @@
                   </li>
                   {{#unless (eq backend.type "cubbyhole")}}
                   <li class="action">
-                    <ConfirmAction
-                      @buttonClasses="link is-destroy"
-                      @confirmTitle="Disable engine?"
-                      @confirmMessage="Any data in this engine will be permanently deleted."
-                      @confirmButtonText="Disable"
-                      @onConfirmAction={{perform disableEngine backend}}
+                    <c.Message
+                      @id={{backend.id}}
+                      @triggerText="Disable"
+                      @message="Any data in this engine will be permanently deleted."
+                      @title="Disable engine?"
+                      @onConfirm={{perform disableEngine backend}}
                       data-test-engine-disable="true"
-                    >
-                      Disable
-                    </ConfirmAction>
+                      />
                    </li>
                    {{/unless}}
                  {{#if item.updatePath.isPending}}
                    <li class="action">
                      <button disabled type="button" class="link button is-loading is-transparent"></button>
```

Comment:
I noticed this empty button and wondered what it was for, I also noticed there is one above with the text `loading` inside so I'm assuming its similar, I know its not part of your changeset but do this one need a `loading` inside it also? (Or maybe the other one needs `loading` removing?)

Reply:
good eye! i found several other [examples of buttons](https://github.com/hashicorp/vault/blob/master/ui/app/templates/partials/role-aws/popup-menu.hbs#L19-L26) with the `is-loading` class and i think you're right -- this button should have `loading` inside it. thanks for noticing!